### PR TITLE
update and delete notes

### PR DESCRIPTION
**Pull request by Alex90Jennings**

### CODE REVIEW

File: src/components/profile/Note.js [hunk 1]
```diff
@@ -1,9 +1,74 @@
...
+  const [noteEdit, setNoteEdit] = useState({ content: '' });
+  const [checkIfEditing, setCheckIfEditing] = useState(false)
+
+  const deleteNoteHandler = (event, noteId) => {
+    event.preventDefault();
+    if(isDeleting){
+      client
+        .delete(`/note/${noteId}`)
+        .then(() => setIsDeleting(false))
+        .catch((err) => console.error(err.response))
```
Comment (dogezen) [inline src/components/profile/Note.js:22]: is there a reason why the catch is not at the end of the `then` chain?

File: src/components/profile/Note.js [hunk 2]
```diff
@@ -1,9 +1,74 @@
...
+					onChange={(e) => handleChange(e)}
+					size={30}
+				></textarea>
+			)) || note.content}
+      <div className='three-column-expand-one'>
+        {note.isEdited && (		
+        <div className='time'>
+						<p>Edited {formatDate(Date.parse(note.updatedAt))}</p>
+				</div>)}
+        {loggedInUser.role === 'TEACHER' && (<Button className='delete-btn note-button' size='small' variant='contained' color={isDeleting ? 'error' : 'primary'} onClick={(event) => deleteNoteHandler(event, note.id)}>{isDeleting ? 'Confirm': 'Delete'}</Button>)}
```
Comment (dogezen) [inline src/components/profile/Note.js:69]: code formatting - let's make sure indentations are good here please

File: src/components/profile/Note.js (hunk 1 above)
Comment (sigristarisa) [reply src/components/profile/Note.js]: There is no reason – didn't know ``.catch``should be in the end

File: src/components/profile/Note.js (hunk 2 above)
Comment (Alex90Jennings) [reply src/components/profile/Note.js]: changed it in VS, I hope it looks better this time

File: src/components/profile/Note.js (hunk 1 above)
Comment (Alex90Jennings) [reply src/components/profile/Note.js]: moved the catch to the end of the chain

**Review by dogezen — APPROVED** [review]

**Review by sigristarisa — APPROVED** [review]

**Pull request merged**